tests/Passbook/Tests/PassTest.php

```diff
@@ -2,6 +2,8 @@
 
 namespace Passbook\Tests;
 
+date_default_timezone_set('Europe/London');
```

Comment:
Yes there was, the test failed if date.timezone was not set when running the Boarding Pass test I believe. Normally this wouldn't need to be set, but I tested on a fresh php environment where I hadn't set the time and settings up, which caused the test to fail, so included this as a fix to anyone else that might have the same issue.